portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -106,6 +114,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     response.sendRedirect("/index.html");
   }
 
+  private boolean doesLimitExceeded(int limit) {
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.ASCENDING);
```

Comment:
Why do we need to sort them in order to sum up the sizes?

Reply:
Actually we don't need. I just copy paste from previous query. I will fix it by making query without sorting.